integration-test/common/src/test/java/org/apache/uniffle/test/ShuffleReadWriteBase.java

```diff
@@ -151,7 +151,7 @@ public static ShuffleDataResult readShuffleData(
       SegmentSplitter segmentSplitter) {
     RssGetShuffleIndexRequest rgsir =
         new RssGetShuffleIndexRequest(
```

Comment:
Why not creating the default RssGetShuffleIndexRequest construct with default retryMax=1, that will not effect the previous test cases

Reply:
Done.